front/src/routes/scene/edit-scene/EditScenePage.jsx

```diff
@@ -60,6 +60,9 @@ const EditScenePage = ({ children, ...props }) => (
               <button onClick={props.startScene} class="btn btn-sm btn-primary ml-2">
                 <Text id="editScene.startButton" /> <i class="fe fe-play" />
               </button>
+              <button onClick={props.duplicateScene} disabled={props.saving} className="btn btn-sm btn-warning ml-2">
+                <Text id="editScene.duplicateButton" /> <i className="fe fe-plus" />
```

Comment:
Why not using a "copy" icon ? 

Try with `fe fe-copy`

https://feathericons.com/?query=copy

Reply:
It's a mistake on my side, oops :)